tests/models/test_position_encoding.py

```diff
@@ -0,0 +1,17 @@
+from nowcasting_utils.models.position_encoding import encode_position
+import pytest
+
+def test_fourier_encoding():
+    pass
```

Comment:
todo?

Reply:
Just wanted to get more thoughts on the design before I actually finished this, incase we want to move it elsewhere, I can make it more simplified, etc.